newIDE/app/src/UI/Notification/NotificationList.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+import { Trans } from '@lingui/macro';
+import { type Notification } from '../../Utils/GDevelopServices/Notification';
+import { List, ListItem } from '../List';
+import NotificationListItem from './NotificationListItem';
+import LeftLoader from '../LeftLoader';
+import { Column, Line } from '../Grid';
+import Text from '../Text';
+import FlatButton from '../FlatButton';
+import TextButton from '../TextButton';
+import ScrollView from '../ScrollView';
+
+type Props = {|
+  notifications: Notification[],
+  onMarkAllAsRead: () => Promise<void>,
+  canMarkAllAsRead: boolean,
+  onMarkNotificationAsSeen: Notification => Promise<void>,
+  onCloseNotificationList: () => void,
+|};
+
+const NotificationList = ({
+  notifications,
+  onMarkAllAsRead,
+  canMarkAllAsRead,
+  onMarkNotificationAsSeen,
+  onCloseNotificationList,
+}: Props) => {
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [showAll, setShowAll] = React.useState<boolean>(false);
+
+  const markAllAsRead = React.useCallback(
+    async () => {
+      setIsLoading(true);
+      try {
+        await onMarkAllAsRead();
+      } catch (error) {
+        console.error(
```

Comment:
show an Alert here?

Reply:
I'm not sure: worst case scenario it does not change anything. I think it's a bit scary to show an alert for this